jinja2/includes/icons/general/frown.svg

```diff
@@ -0,0 +1,3 @@
+<svg class="icon icon-frown" xmlns="http://www.w3.org/2000/svg" width="24" height="28" viewBox="0 0 24 28" aria-hidden="true">
```

Comment:
I couldn't find this referenced in the PR. It's OK to keep it, unless you don't plan to use it.

Reply:
This will come into play in the follow on PRs